"#import os" causes recognition by libmagic as "ObjectiveC source" and not emitting script shebang dependency
--- d-feet-0.3.16/src/d-feet.in.orig	2021-05-07 10:46:33.368616300 +0200
+++ d-feet-0.3.16/src/d-feet.in	2021-05-07 20:14:40.551616088 +0200
@@ -20,17 +20,6 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 
-#import os
-#gi_typelib_path = ["@pkglibdir@/girepository-1.0",]
-#if 'GI_TYPELIB_PATH' in os.environ:
-#    gi_typelib_path.append(os.environ['GI_TYPELIB_PATH'])
-#os.environ['GI_TYPELIB_PATH'] = ":".join(gi_typelib_path)
-
-#ld_library_path = ["@pkglibdir@",]
-#if 'LD_LIBRARY_PATH' in os.environ:
-#    ld_library_path.append(os.environ['LD_LIBRARY_PATH'])
-#os.environ['LD_LIBRARY_PATH'] = ":".join(ld_library_path)
-
 import os
 import sys
 sys.path.insert(1, '@pythondir@')
